### PR TITLE
feat: add user registration/login 

### LINKED ISSUE #420: [FEATURE]: Connect user registration and login with backend
### Detailed description

As there is a backend now we can add the login or registration info to the database

### Context

We can collect the user info

### Possible implementation

_No response_

### Additional information

I would like to work on it under gssoc23

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by suzy-g38**
## Related Issue

- Information about the related issue

Closes: #420

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/UniKonf/vibey/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by suzy-g38 — COMMENTED** [review]
changes for generate-token

File: server/pnpm-lock.yaml [hunk 1]
```diff
@@ -5,9 +5,9 @@ settings:
   excludeLinksFromLockfile: false
 
 dependencies:
-  body-parser:
```
Comment (Deepu178) [inline server/pnpm-lock.yaml:8]: is there any reason for removing body-parser?

Comment (suzy-g38) [reply server/pnpm-lock.yaml:8]: Yes, I didn't use body-parser in the backend that's why

Comment (Deepu178) [reply server/pnpm-lock.yaml:8]: got it so we don't need to install another dependency of body-parser for Express 4.16+ if I'm getting it right.

Comment (suzy-g38) [reply server/pnpm-lock.yaml:8]: > got it so we don't need to install another dependency of body-parser for Express 4.16+ if I'm getting it right.

hmm...not at least now

**Pull request merged**